kw_sdk.go

```diff
@@ -0,0 +1,102 @@
+package main
+
+//nolint:godox
+// TODO: figure out if it's worth to move this to a dedicated library
```

Comment:
Not blocking, I just wonder if we can move that to a sub-dir of our SDK :thinking: